src/views/UserList.vue

```diff
@@ -1,110 +1,176 @@
 <template>
   <div>
     <NavList />
-    <br>
-    <br>
-    <br>
-    <br>
-    <br>
-    <div>
+
+    <div class="buttons">
       <a class="Cadastro">Cadastro</a>
       <a class="Listagem">Listagem</a>
-      <br>
-      <br>
-      <br>
       <button class="btn">Cadastrar Novo Usuario</button>
     </div>
     <div class="user">
-      <br>
       <br>
       <hr>
-     <a id="cpf">CPF</a>
+      <a id="cpf">CPF</a>
       <a id="FullName">Nome Completo</a>
-      <br>  
-      <br>
       <hr>
     </div>
-    <div>
+    <div id="renderResult">
       <br>
-      <div id="pagination">
-      <Pagination/>
-      <br>
-      <br>
-      </div>
-      </div>
-    </div>
+      <ul v-for="item in pageOfItems" :key="item.id">
     
+          <li id="content-cpf" >{{ item.cpf | VMask(mask)}} <br></li>
+        
+          <br>
+          <br>
```

Comment:
"Note: Do not use <br> to create margins between paragraphs; wrap them in [<p>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p) elements and use the [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) [margin](https://developer.mozilla.org/en-US/docs/Web/CSS/margin) property to control their size."

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br